crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -872,7 +853,7 @@ impl AuthorityPerEpochStore {
             .set(randomness_manager.clone())
             .is_err()
         {
-            error!("`set_randomness_manager` called more than once; this should never happen");
+            error!("BUG: `set_randomness_manager` called more than once; this should never happen");
```

Comment:
should we panic then?

Or in other words, if this is called more than once would it result in correctness issues or not?

Reply:
I think it could result in liveness issues but not correctness issues, which to me seemed not worth a `panic` but I don't have a strong feeling about it if you think panic is better. It would be nice if we had a log level above `error` that would show up for us in a report somewhere about definite bugs, so that we didn't have to crash binaries to notice something is wrong... something for the long term "nice to have" list

Reply:
i'd love a debug assert at least